test/unit/collection-view.spec.js

```diff
@@ -28,6 +28,34 @@ describe('collection view', function() {
   // Collection View Specs
   // ---------------------
 
+  describe('when a collection view is DOM', function() {
+    beforeEach(function() {
+      this.$fixtureEl = $('<div id="fixture-collectionview"></div>');
```

Comment:
something like
```
this.$fixtureEl = $('<div id="fixture-collectionview"><span id="el1">1</span></div>');

this.view1 = new this.ChildView({el: this.$fixtureEl.find('#el1')[0]});
```
then
```
this.collectionView.addChildView(this.view1, 0);
```
although I haven't tried that addChildView nor do I know if it will work